demisto_sdk/commands/common/content/objects/pack_objects/abstract_pack_objects/yaml_content_object.py

```diff
@@ -87,7 +87,7 @@ def from_version(self) -> Union[Version, LegacyVersion]:
             1. Version object - https://github.com/pypa/packaging
             2. Attribute info - https://xsoar.pan.dev/docs/integrations/yaml-file#version-and-tests
         """
-        return parse(self.get("fromversion", DEFAULT_CONTENT_ITEM_FROM_VERSION))
+        return Version(self.get("fromversion", DEFAULT_CONTENT_ITEM_FROM_VERSION))
 
     @property
     def to_version(self) -> Union[Version, LegacyVersion]:
```

Comment:
```suggestion
    def to_version(self) -> Version:
```
And the same for the previous function.